config/application.rb

```diff
@@ -10,6 +10,7 @@
 require "action_view/railtie"
 require "action_cable/engine"
 require "sprockets/railtie"
+require "active_storage"
```

Comment:
What are we using ActiveStorage for now?

Reply:
It's a file getter